windows-observ-lib/panels.libsonnet

```diff
@@ -449,5 +449,327 @@ local utils = commonlib.utils;
+                                + fieldOverride.byName.withPropertiesFromOptions(
+                                  table.standardOptions.withUnit('ops')
+                                  + table.standardOptions.color.withMode('continuous-BlPu')
+                                ),
+                              ])
+                              + table.queryOptions.withTransformationsMixin(
+                                [
+                                  {
+                                    id: 'joinByLabels',
+                                    options: {
+                                      join: [
+                                        'instance',
+                                      ],
+                                      value: 'bind_method',
+                                    },
+                                  },
+                                  {
+                                    id: 'filterFieldsByName',
+                                    options: {
+                                      include: {
+                                        pattern: 'instance|digest|ds_client|ds_server|external|fast|ldap|negotiate|ntlm|simple',
+                                      },
+                                    },
+                                  },
+                                  {
+                                    id: 'organize',
+                                    options: {
+                                      renameByName:
+                                        {
+                                          instance: 'Instance',
+                                          digest: 'Digest',
+                                          ds_client: 'DS_client',
+                                          ds_server: 'DS_server',
+                                          external: 'External',
+                                          fast: 'Fast',
+                                          ldap: 'LDAP',
+                                          negotiate: 'Negotiate',
+                                          ntlm: 'NTLM',
+                                          simple: 'Simple',
+                                        },
+                                    },
+                                  },
+                                  {
+                                    id: 'reduce',
+                                    options: {
+                                      includeTimeField: false,
+                                      mode: 'reduceFields',
+                                      reducers: [
+                                        'last',
+                                      ],
+                                    },
+                                  },
+                                ]
+                              ),
+      intrasiteReplicationTraffic: commonlib.panels.generic.timeSeries.base.new(
```

Comment:
For this panel and the intersite version, do they need the min, max, mean legend calcs?

Reply:
Same for inboundReplicationUpdates

Reply:
I added them in for now, I know previously I did not have them in, but I want to see what Grafana will say to this. The reason for the switch is I think having a calculated mean would be really helpful, and as far as i've seen whenever there is a calculated mean, there is always a min and max that follows it.

Reply:
Fair enough to see what they say, but at that rate they may prefer a stat panel instead, based on feedback I got during spec review at one point when I had a similar situation ... just something to keep in mind